spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -247,6 +256,60 @@ public void updateItem(String groupId, Item item, List<String> ignoreFields, Lis
             throw new ZoteroConnectionException("Could not update item.", e);
         }
     }
+    
+    
+    
+    /**
+     * This method makes a batch request call to Zotero to update items
+     * 
+     * @param groupId      group id of citations
+     * @param items        items that have to be updated
+     * @param ignoreFields fields that are not necessary while updating citations
+     * 
+     * @return ZoteroUpdateItemsStatuses returns items statuses
+     */
+    @Override
+    public ZoteroUpdateItemsStatuses batchUpdateItems(String groupId, List<Item> items,
+            List<List<String>> ignoreFieldsList, List<List<String>> validCreatorTypesList)
+            throws ZoteroConnectionException {
+        List<ItemCreationResponse> responses = new ArrayList<>();
+        int totalItems = items.size() - 1;
+        int itemsDone = 0;
+        List<String> itemsKeys = new ArrayList<>();
+
+        while (totalItems >= itemsDone) {
+            int count = 0;
+            List<JsonNode> dataAsJsonArray = new ArrayList<>();
+            for (; itemsDone <= totalItems && count < ZOTERO_BATCH_UPDATE_LIMIT; count++, itemsDone++) {
+                dataAsJsonArray.add(createDataJson(items.get(itemsDone), ignoreFieldsList.get(itemsDone),
+                        validCreatorTypesList.get(itemsDone), false));
+                itemsKeys.add(items.get(itemsDone).getKey());
+            }
+
+            ObjectMapper mapper = new ObjectMapper();
+            ArrayNode arrayNode = mapper.createArrayNode();
+            arrayNode.addAll(dataAsJsonArray);
+            String jsonArrayString = null;
+            try {
+                jsonArrayString = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(arrayNode);
+            } catch (JsonProcessingException e) {
+                throw new ZoteroConnectionException("Could not serialize data.", e);
```

Comment:
I would throw a different exception here since this is not really a zotero or connection issue.